louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
+
+ a) "1" == 1
+  true
+
+ b) "1" === 1
+  false
+
+ c) x == "x"
+  true
```

Comment:
Remember that we're evaluating variable x against 'x' in this situation, where x can contain any value. Unless that value is 'x', this equation will always return false.